docs/visualization/rgb.rst

```diff
@@ -91,3 +91,157 @@ of the galaxies show up. Compare with Fig. 1 of `Lupton et al. (2004)`_ or the
 
 
 .. _SDSS Skyserver image: https://skyserver.sdss.org/dr13/en/tools/chart/navi.aspx?ra=313.12381&dec=-5.74611
+
+
+.. _astropy-visualization-rgb-log-linear:
+
+RGB images using logarithmic or linear stretching
+=================================================
+
+Numerous other methods for generating composite RGB images are possible.
+Two alternative choices are using a linear or logarithmic stretch, combined
+with optional data clipping and normalization (e.g., as often used in DS9 or
+other data viewers).
+
+The image stretching and normalization methods for single images are
+demonstrated in :ref:`astropy-visualization-stretchnorm`.
+These scaling are extended to the generation of RGB images in the
+convenience wrapper functions :func:`~astropy.visualization.make_log_rgb` and
+:func:`~astropy.visualization.make_linear_rgb`.
+
+As with :func:`~astropy.visualization.make_lupton_rgb`, the three images must
+be aligned, with the same size and pixel scales. For both functions,
+``minimum`` and ``maximum`` specify the normalization (if any) and the
+black and white levels, respectively. Both parameters can be either a scalar,
+applying the same normalization to each filter, or can be a length-3 array
+separately specifying the per-filter minimum or maximum values.
+Logarithmic scaling also requires ``scalea``, with the stretch of values
+calculated as
+
+.. math::
+
+    y = \frac{\log{(a x + 1)}}{\log{(a + 1)}}
+
+Following the above example, we generate a composite RGB image using the
+``g``, ``r``, ``i`` SDSS frames around the Hickson 88 group,
+now using a logarithmic scaling.
+
+.. plot::
+   :context: reset
+   :include-source:
+   :align: center
+
+    import numpy as np
+    import matplotlib.pyplot as plt
+    from astropy.visualization import make_log_rgb
+    from astropy.io import fits
+    from astropy.utils.data import get_pkg_data_filename
+
+    # Read in the three images downloaded from here:
+    g_name = get_pkg_data_filename('visualization/reprojected_sdss_g.fits.bz2')
+    r_name = get_pkg_data_filename('visualization/reprojected_sdss_r.fits.bz2')
+    i_name = get_pkg_data_filename('visualization/reprojected_sdss_i.fits.bz2')
+    g = fits.open(g_name)[0].data
+    r = fits.open(r_name)[0].data
+    i = fits.open(i_name)[0].data
```

Comment:
Examples should use `fits.getdata(i_name)` instead, to avoid leaking file descriptors.